lib/spack/spack/binary_distribution.py

```diff
@@ -123,7 +125,7 @@ def write_buildinfo_file(prefix, rel=False):
     buildinfo['buildpath'] = spack.store.layout.root
     buildinfo['relocate_textfiles'] = text_to_relocate
     buildinfo['relocate_binaries'] = binary_to_relocate
-    filename = buildinfo_file_name(prefix)
+    filename = buildinfo_file_name(workdir)
```

Comment:
What issue does this fix?

Reply:
It clarifies things for me to distinguish the working copy from the original prefix. The buildinfo file is written to the working copy.

Reply:
I added it to calculate the relative install prefix.